lib/plugins/track.js

```diff
@@ -189,7 +198,7 @@ exports.configure = function configure (config) {
 
```

Comment:
Possibly, if we want to restructure how this section of code works ... I'll give it a shot.